boilerplate/flyte/end2end/run-tests.py

```diff
@@ -277,6 +290,12 @@ def run(
 @click.argument("flytesnacks_release_tag")
 @click.argument("priorities")
 @click.argument("config_file")
+@click.argument(
+    "cluster_pool_name",
+    required=False,
+    type=str,
+    default=None,
+)
```

Comment:
just checking: did you give running this a try via CLI with no `cluster_pool_name` arg and it was working as intended?

Reply:
yes